package.json

```diff
@@ -38,7 +38,7 @@
     "postcss": "^8.4.4",
     "pretend": "^3.1.0",
     "tslib": "^2.3.1",
-    "@elastosfoundation/did-js-sdk": "^2.2.4"
+    "@elastosfoundation/did-js-sdk": "^2.2.7"
```

Comment:
We can put 2.2.9 here since this is a dynamic version number and it will get the latest minor.